app/controllers/shared/api.php

```diff
@@ -362,7 +362,8 @@
     ->inject('queueForUsage')
     ->inject('dbForProject')
     ->inject('mode')
-    ->action(function (App $utopia, Request $request, Response $response, Document $project, Document $user, Event $queueForEvents, Messaging $queueForMessaging, Audit $queueForAudits, Delete $queueForDeletes, EventDatabase $queueForDatabase, Build $queueForBuilds, Usage $queueForUsage, Database $dbForProject, string $mode) use ($databaseListener) {
+    ->inject('hasDevelopmentKey')
```

Comment:
Very bad name. We never use this kind of question form for resources. Either we use a resource as a resource for DI or we find a different way to do this.